docs/contact.rst

```diff
@@ -10,6 +10,9 @@ making our products better, or pretty much anything else, reach out!
     * https://github.com/astronomy-commons/lsdb/issues/new
 * Start a new github discussion
     * https://github.com/astronomy-commons/lsdb/discussions/new/choose
+* Join the LSST Community forum
```

Comment:
If you have questions about data from Rubin Observatory, the best place to ask is: